src/extension/tools/node/findTextInFilesTool.tsx

```diff
@@ -126,16 +128,20 @@ export class FindTextInFilesTool implements ICopilotTool<IFindTextInFilesToolPar
 		}
 	}
 
-	private async searchAndCollectResults(query: string, isRegExp: boolean, patterns: vscode.GlobPattern[] | undefined, maxResults: number, token: CancellationToken): Promise<vscode.TextSearchResult2[]> {
+	private async searchAndCollectResults(query: string, isRegExp: boolean, patterns: vscode.GlobPattern[] | undefined, maxResults: number, includeIgnoredFiles: boolean | undefined, token: CancellationToken): Promise<vscode.TextSearchResult2[]> {
+		const findOptions = {
```

Comment:
If you write this as `const findOptions: vscode.FindTextInFilesOptions2` instead of casting, then you will get an error if the object includes a key that isn't a correct prop in that object.

Reply:
You are right. Updated.